infrastructure/aat.tfvars

```diff
@@ -0,0 +1,2 @@
+capacity = "1"
```

Comment:
let's keep for now ... but don't think we need this any more

Reply:
I have retained the same variables as ia-case-api